includes/create-theme/theme-locale.php

```diff
@@ -28,6 +28,72 @@ private static function escape_text_content( $string ) {
 
 		$string = addcslashes( $string, "'" );
 
```

Comment:
Perhaps there's enough going on here this should have it's own utility class?

Reply:
I tried splitting this out in [bf8ff55](https://github.com/WordPress/create-block-theme/pull/719/commits/bf8ff55e9629f9bf31317bf18e806f5d086b61ef).